Modelica/Math/package.mo

```diff
@@ -1385,7 +1385,7 @@ has a unique solution.
 </html>"));
   end equalityLeastSquares;
 
-  function LU "LU decomposition of square or rectangular matrix"
+  pure function LU "LU decomposition of square or rectangular matrix"
```

Comment:
Not sure if this is pure or not. If it is translated on one machine and run on another, calls to LU during translation time might give very different results from runtime (because there are multiple solutions and there could be different lapack implementations involved; or the same one on different CPU architectures).

Reply:
Same argument as you remarked for getPid: This is pure if it runs only during simulation time.